aml_vision/image-classification/README.md

```diff
@@ -0,0 +1,81 @@
+# AzureML Foundational Vision Models Fine-Tuning Examples
```

Comment:
Can we change the title to "Foundational Vision Models Fine-Tuning with ACPT and ONNX Runtime"